### ISSUE TITLE
[Bug] Controller applet constantly appearing in random intervals for no reason

### ISSUE THREAD

**Issue by Noah-HalfSquid** [Ryujinx/Ryujinx#5512]
### Description of the issue

So I'm trying to play Pikmin 4, and after anywhere between 5 seconds to 5 minutes (completely random) it'll give a popup saying "application requests exactly 1 player" even though I only have 1 controller slot actually used (even handheld is turned off) so I'm not sure why I keep getting these popups.

### Reproduction steps

I'm using the version 1.01 nsp on vulkan with otherwise default settings.

### Log file

[Ryujinx_1.1.0-ldn3.0.1_2023-07-31_20-54-34.log](https://github.com/Ryujinx/Ryujinx/files/12224141/Ryujinx_1.1.0-ldn3.0.1_2023-07-31_20-54-34.log)

### OS

Windows 10

### Ryujinx version

ldn3.0.1 (or 1.1.0)

### Game version

1.01

**Comment by Noah-HalfSquid** [comment]
![image](https://github.com/Ryujinx/Ryujinx/assets/119549393/ddbf0fdb-c9cd-44bd-90ea-57b14d64a4ed)
Issue still occurs on the latest official build

**Comment by GamerzHell9137** [comment]
That's happening because your controller is disconnecting.

**Comment by Noah-HalfSquid** [comment]
> That's happening because your controller is disconnecting. You gotta fix that.

So, I did some messing around, and I fixed the issue with my controller disconnecting, but it still does that. And if I'm tabbed out while it tries to display that popup, ryujinx just freezes. The console shows absolutely nothing of use. I genuinely have no clue what could be causing this, it got worse and last time I tested it happened like 10 times in the span of a minute, with my controller never disconnecting. It's making the game unplayable at this point and I don't know what to do.
